### PR TITLE
chore: a bunch of dependency updates

### PR DESCRIPTION
**Pull request by ableuler**
One of them patches a vulnerability.

/deploy

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,11 @@
-FROM python:3.7-alpine
```
Comment (rokroskar) [inline Dockerfile:1]: why this change?

Comment (olevski) [reply Dockerfile:1]: @ableuler why are we changing the base image? Was this also a security vulnerability?

Comment (ableuler) [reply Dockerfile:1]: Dependabot upgraded the cryptography package which doesn't play well with the alpine based image (we'd have to add a rust compiler). After switching to the debian based python image the package is pip-installable again. But I quickly wanted to try with the `-slim` base image and compare sizes, should have labeled it as WIP.

Comment (ableuler) [reply Dockerfile:1]: https://github.com/pyca/cryptography/issues/5771

Comment (ableuler) [reply Dockerfile:1]: The slim image is actually smaller than the resulting alpine image with the necessary installs.

Comment (rokroskar) [reply Dockerfile:1]: double win 🥇

**Comment by ableuler** [comment]
anybody feels like approving the PR? :)

**Review by olevski — APPROVED** [review]

**Comment by rokroskar** [comment]
ah sorry, I was waiting for acceptance tests to finish.. and then it was lunchtime 👍 😄
